### PR TITLE
Switched inverse function to run pseudo-inverse, to account for correlated and 0 variance variables

### PR DESCRIPTION
**Pull request by Rambatino**
Switched to pinv from inv

https://docs.scipy.org/doc/numpy/reference/generated/numpy.linalg.pinv.html

It uses: 

https://en.wikipedia.org/wiki/Moore%E2%80%93Penrose_pseudoinverse

With thanks to @jhofmeyr

### CODE REVIEW

**Review by vpolimenov — APPROVED** [review]

File: tests/test_kruskals.py [hunk 1]
```diff
@@ -121,3 +121,26 @@ def test_that_direction_is_applied_on_directional_drivers_analysis():
...
+      [1, 0, 2, 9, 5, 4],
+      [1, 0, 2, 9, 5, 4]
+    ])
+
+    arr = np.array([1, 2, 3, 4, 5, 6, 7, 8])
+
+    series = Kruskals.Kruskals(ndarr, arr).driver_score()
+
+    assert series[1] == 0.0
+    assert series[4] == 0.0
```
Comment (xulaus) [inline tests/test_kruskals.py]: New line at end of file

**Review by xulaus — CHANGES_REQUESTED** [review]

File: tests/test_kruskals.py (hunk 1 above)
Comment (Rambatino) [reply tests/test_kruskals.py]: ☺️

File: tests/test_kruskals.py [hunk 2]
```diff
@@ -121,3 +121,26 @@ def test_that_direction_is_applied_on_directional_drivers_analysis():
     series = Kruskals.Kruskals(ndarr, arr).driver_score_to_series(True)
 
     assert (series.values < 0).any()
+
```
Comment (xulaus) [inline tests/test_kruskals.py:124]: Also I think this should be a double newline

**Comment by xulaus** [comment]
#whitespacepolice

File: tests/test_kruskals.py (hunk 2 above)
Comment (Rambatino) [reply tests/test_kruskals.py:124]: Not given that the others aren't. We can correct in a different PR if you feel strongly

**Review by xulaus — APPROVED** [review]

**Pull request merged**